### PR TITLE
OTAManager: Pick the right binaries on Kindles running on the Bellatrix platform

### PR DESCRIPTION
**Pull request by NiLuJe**
Re: https://github.com/koreader/koreader/pull/4539#issuecomment-1411707712

- - -
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/10068)

### CODE REVIEW

**Comment by NiLuJe** [comment]
(Gratuitous fix to the unespaced `.` in the i.MX pattern, too ;p).

**Comment by NiLuJe** [comment]
For archeological purposes, PW5 cpuinfo:

```
┌─(ROOT@kindle:pts/0)──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────(~)─┐
└─(3.02:58%:23:58:94%:#)── cat /proc/cpuinfo                                                                                                                                                                                                                                                                                                           ──(Wed, Feb 01)─┘
processor       : 0
Processor       : ARMv7 Processor rev 4 (v7l)
model name      : ARMv7 Processor rev 4 (v7l)
BogoMIPS        : 21.65
Features        : half thumb fastmult vfp edsp neon vfpv3 tls vfpv4 idiva idivt vfpd32 lpae evtstrm aes pmull sha1 sha2 crc32 
CPU implementer : 0x41
CPU architecture: 7
CPU variant     : 0x0
CPU part        : 0xd03
CPU revision    : 4

processor       : 1
Processor       : ARMv7 Processor rev 4 (v7l)
model name      : ARMv7 Processor rev 4 (v7l)
BogoMIPS        : 21.65
Features        : half thumb fastmult vfp edsp neon vfpv3 tls vfpv4 idiva idivt vfpd32 lpae evtstrm aes pmull sha1 sha2 crc32 
CPU implementer : 0x41
CPU architecture: 7
CPU variant     : 0x0
CPU part        : 0xd03
CPU revision    : 4

Hardware        : MT8110 Bellatrix device
Revision        : 0000
Serial          : <16chars hex>

**Comment by Frenzie** [comment]
Rebase please. ;-)

File: frontend/ui/otamanager.lua [hunk 1]
```diff
@@ -79,8 +79,12 @@ local arch = jit.arch
 
 function OTAManager:getOTAModel()
     if Device:isAndroid() then
-        if arch == "x86" then
+        if arch == "arm64"
+            return "android-arm64"
+        elseif arch == "x86" then
             return "android-x86"
+        elseif arch == "x86_64" then
+            return "android-x86_64"
```
Comment (Frenzie) [inline frontend/ui/otamanager.lua:87]: A tiny bit of future-proofing.

**Pull request merged**